src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -56,77 +56,26 @@
             "description": "A Razor TagHelper Attribute"
           },
           {
-            "id": "razorTagHelperTransition",
-            "description": "A Razor TagHelper transition"
+            "id": "razorTransition",
+            "description": "A Razor transition"
           },
           {
-            "id": "razorTagHelperColon",
+            "id": "razorDirectiveAttribute",
+            "description": "A Razor Directive Attribute"
+          },
+          {
+            "id": "razorDirectiveColon",
             "description": "A colon between directive attribute parameters"
           }
         ],
-        "semanticTokenStyleDefaults": [
-          {
-            "selector": "razorTagHelperElement",
-            "scope": [ "razor.taghelpers.element" ],
-            "light": {
-              "foreground": "#800080",
-              "fontStyle": "bold"
-            },
-            "dark": {
-              "foreground": "#009696",
-              "fontStyle": "bold"
-            }
-          },
+        "semanticTokenScopes": [
           {
-            "selector": "razorTagHelperAttribute",
-            "scope": [ "razor.taghelpers.attribute" ],
-            "light": {
-              "foreground": "#800080",
-              "fontStyle": "bold"
-            },
-            "dark": {
-              "foreground": "#009696",
-              "fontStyle": "bold"
-            }
-          },
-          {
-            "selector": "razorTagHelperColon",
-            "scope": [ "razor.taghelper.colon"],
-            "light": {
-              "foreground": "#C586C0"
-            },
-            "dark": {
-              "foreground": "#C586C0"
-            }
-          },
-          {
-            "selector": "razorTransition",
-            "scope": [ "razor.transition"],
-            "light": {
-              "foreground": "#C586C0"
-            },
-            "dark": {
-              "foreground": "#C586C0"
-            }
-          },
-          {
-            "selector": "razorDirectiveAttribute",
-            "scope": [ "razor.directive.attribute"],
-            "light": {
-              "foreground": "#C586C0"
-            },
-            "dark": {
-              "foreground": "#C586C0"
-            }
-          },
-          {
-            "selector": "razorDirectiveAttributeParameter",
-            "scope": [ "razor.directive.parameter"],
-            "light": {
-              "foreground": "#C586C0"
-            },
-            "dark": {
-              "foreground": "#C586C0"
+            "scopes": {
+              "razorTagHelperElement": [ "entity.name.class.element.taghelper" ],
```

Comment:
We now have to adhere to existing scopes to get our  pretty colors. Still working on also getting bold for element and attribute.